data-reporting-service/investigation-service/src/test/java/gov/cdc/etldatapipeline/investigation/InvestigationDataProcessingTests.java

```diff
@@ -38,6 +48,7 @@ public class InvestigationDataProcessingTests {
     private static final String CONFIRMATION_TOPIC = "confirmationTopic";
     private static final String OBSERVATION_TOPIC = "observationTopic";
     private static final String NOTIFICATION_TOPIC = "notificationTopic";
+    private static final String NOTIFICATIONS_TOPIC = "notificationsTopic";
```

Comment:
NOTIFICATION_TOPIC on line 50 will be removed as part of the observation_notification_ids, correct?